src/Common/tests/System/Security/Cryptography/AlgorithmImplementations/AES/DecryptorReusability.cs

```diff
@@ -13,6 +13,10 @@ public static class DecryptorReusabilty
         [Fact]
         public static void TestDecryptorReusability()
         {
+            // See https://github.com/dotnet/corefx/issues/18903 for details
+            if (!ShouldDescriptorBeReusable())
+                return;
```

Comment:
We should do this with a ConditionalFact instead, e.g.
```C#
[ConditionalFact(nameof(ShouldDescriptorBeReusable))]
```
That way it'll show as a skipped test in the output if the check fails.